builtin/providers/azurerm/resource_arm_loadbalancer.go

```diff
@@ -172,7 +177,18 @@ func resourecArmLoadBalancerRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("resource_group_name", id.ResourceGroup)
 
 	if loadBalancer.LoadBalancerPropertiesFormat != nil && loadBalancer.LoadBalancerPropertiesFormat.FrontendIPConfigurations != nil {
-		d.Set("frontend_ip_configuration", flattenLoadBalancerFrontendIpConfiguration(loadBalancer.LoadBalancerPropertiesFormat.FrontendIPConfigurations))
+		ipconfig := flattenLoadBalancerFrontendIpConfiguration(loadBalancer.LoadBalancerPropertiesFormat.FrontendIPConfigurations)
+		d.Set("frontend_ip_configuration", ipconfig)
+
+		for _, config := range ipconfig {
+			cfg := config.(map[string]interface{})
+			if priv_ip, ok := cfg["private_ip_address"]; ok {
```

Comment:
That had occurred to me too. I opted to iterate over the flattened list so there would be a single piece of code accessing ``loadBalancer.LoadBalancerPropertyFormat.FrontendIPConfigurations.*.FrontendIPConfigurationPropertiesFormat.PrivateIPAddress``, just in case the API changes in the future.

I can definitely switch it up when I get back to a keyboard tomorrow if you'd like though!